packages/orbit-design-tokens/docs/SCHEMA.md

```diff
@@ -97,7 +97,7 @@ For instance, if we want to deprecate the `component.button.large.height` design
           "value": "{foundation.size.extra-large}",
           "deprecated": true,
           "deprecated-replace": "{optional.alias.to.specify.the.replacement.token}",
-          "deprecated-version": "1.0.0"
+          "deprecated-version": "6.0.0"
```

Comment:
What does this change means? I saw it on every tokens definition, but I don't understand it

Reply:
Yeah, that's actually showing the version in which tokens are supposed to be removed :) so next major after the official v5 release